eZ/Publish/Core/Search/Common/Indexer.php

```diff
@@ -85,12 +81,11 @@ protected function getContentDbFieldsStmt(array $fields)
      */
     protected function getContentLocationIds($contentObjectId)
```

Comment:
```suggestion
    protected function getContentLocationIds($contentObjectId): array
```

Reply:
Actually, both `getContentLocationIds` one and `logWarning` are no longer used. They were used in dedicated command for reindexing of Solr and Elasticsearch which both no longer exist.
Dropped via 1f08f69. Thanks for making me take a closer look ;)